src/schemas/Comparison.json

```diff
@@ -7,8 +7,7 @@
     {
       "properties": {
         "comparison": {
-          "type": "string",
-          "const": "superset"
+          "type": "string"
```

Comment:
It would be nice if you could leave the existing strict checks in place, so that if "superset" is the value then the value type is strictly checked. It is possible to a "else" clause for unknown comparisons?

Reply:
I think there's an if/else construct I can use to do that. I'll give it a try.

Reply:
I ended up sticking with "oneOf" and using "not" to handle the unknown comparisons.